### DISCUSSION TITLE
Hello, I am new to programming. What is the easiest and best code editor for multilingual programming?

### DISCUSSION THREAD

**Discussion by 1HHHHHHHHHH1** [community/community#28084] [Other Feature Feedback, Questions, & Ideas]
?

**Comment by zigad** [comment]
Hi! You can start on Youtube. For beginners I recommend Python and then you will see what you like. For Code Editor you can for now use simple text editor or maybe Sublime Text. There are many options there.

If you are also new to Git, you can use my Git Playground repo to try things out without breaking anything: https://github.com/zigad/Playground.

**Comment by baraaj** [comment]
I highly recommend visual studio code

**Comment by zgcwkj** [comment]
> 如果你是在中国我建议你学Java。如果不是，建议学C#，Visual Studio 2022 可以直接上手，官方文档也很齐全！
If you are in China I suggest you learn Java. If not, it is recommended to learn C#,Visual Studio 2022 can be used directly, and the official documents are also very complete!

**Comment by rilysh** [comment]
You don't need an IDE for beginning, start with Notepad++, being a little mature to move on VSCode. As per I assume you're using Windows.

**Reply by cernii** [reply]
Totally agree! There are no common solutions. Each thing in programming is for a special purpose.
Code editor for multilingual programming can be whatever you would like to use and will be comfortable to you.

- notepad++
- atom
- sublime text
- visual studio code

Then when you will feel that you need something more not only editing the code you can try IDEs:

- Jetbrains has many products for different languages and purposes.
- Microsoft Visual Studio for today is the one of the best IDE for C#/VB

**Comment by GustavoLuizO** [comment]
I recommend visual studio code!
